components/dashboard/src/admin/ProjectsSearch.tsx

```diff
@@ -169,9 +170,11 @@ export function ProjectsSearch() {
                         <div className="text-gray-500 dark:text-gray-100 truncate">{p.project.cloneUrl}</div>
                     </div>
                     <div className="flex w-2/12 self-center">
-                        <div className="text-sm w-full text-gray-400 truncate">
-                            {dayjs(p.project.creationTime).fromNow()}
-                        </div>
+                        <Tooltip className="w-fit" content={dayjs(p.project.creationTime).format("MMM D, YYYY")}>
+                            <div className="text-sm w-full text-gray-400 truncate">
+                                {dayjs(p.project.creationTime).fromNow()}
+                            </div>
+                        </Tooltip>
```

Comment:
**suggestion**: I skipped mentioning in https://github.com/gitpod-io/gitpod/pull/15994#pullrequestreview-1266443010 that this is also missing from the teams search in the admin dashboard. Sorry for not clarifying—if that's trivial for you to add could you please add it there, too? Otherwise, we can merge this UX as is an open a follow-up PR.

Reply:
Actually I can't access `admin` route, I can add tooltip but I'll not sure if it's gonna fit or break something else.

Reply:
@tanishq-singh-2301 It's the same area where you've made the same changes for [projects](https://github.com/gitpod-io/gitpod/pull/15994/files#diff-6fa0733647f6ea4dc958623adcead0f2cec8c9b64a26406b7a77b1d20bddd624), [users](https://github.com/gitpod-io/gitpod/pull/15994/files#diff-289e945ddd7cbc2f8d552b92792839d7329dcd6f7001c6600136aee9bb2f32c9), and [workspaces](https://github.com/gitpod-io/gitpod/pull/15994/files#diff-caceff449f575088aad98c86d4e9d600f66db721517693f332beabaf3bd9bec5) in the admin dashboard. However, this should be good to merge as is. Let's wait for someone from `@gitpod-io/engineering-webapp` to chime in for the final approval.